source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/row-actions.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+
+import { Dropdown, Icon } from 'semantic-ui-react';
+import { withTranslations, i18nProps } from '@lib/i18n';
+
+import { attributesFor } from '@data';
+import { ProjectAttributes } from '@data/models/project';
+
+import { withProjectOperations } from '@ui/routes/project/with-project-operations';
+
+export interface IProps {
+  project: JSONAPI<ProjectAttributes>;
+  toggleArchiveProject: (project: JSONAPI<ProjectAttributes>) => void;
+}
+
+class RowActions extends React.Component<IProps & i18nProps> {
+
+  toggleArchivedProject = () => {
+    const { project, toggleArchiveProject } = this.props;
+    toggleArchiveProject(project);
+  }
+
+  render() {
+
+    const { t, project } = this.props;
+    const { dateArchived } = attributesFor(project);
+
+    const dropdownItemText = !dateArchived ?
```

Comment:
this is just a style thing, so... do with this as you wish, but I personally prefer conditions to be as positive as possible. unnecessarily negating things can lead to confusion from future people looking at this.